cmsis/TARGET_CORTEX_A/irq_ctrl_gic.c

```diff
@@ -24,8 +24,6 @@
 
 #include <stddef.h>
 
-#include <cmsis.h>
```

Comment:
Need to fix this, it has real dependency on the target header file (to get gic present macro in), thus I'll look at cmsis build api, and propose a solution)